streams/src/main/java/org/apache/kafka/streams/state/StreamsMetadata.java

```diff
@@ -37,8 +37,8 @@
      * operations.
      */
     public final static StreamsMetadata NOT_AVAILABLE = new StreamsMetadata(new HostInfo("unavailable", -1),
-                                                                            Collections.emptySet(),
-                                                                            Collections.emptySet());
+            Collections.emptySet(),
```

Comment:
nit: move `new HostInfo("unavailable", -1)` to next line to get proper alignment -- also, we use 4 spaces (not 8).